Author: Marcin Kryczek
Description:
 Connection rates are displayed in bytes, but the top bar is graduated in bits.

diff -Naurp iftop.orig/ui.c iftop/ui.c
--- iftop.orig/ui.c	2008-09-08 01:53:08.000000000 +0200
+++ iftop/ui.c	2009-01-27 17:44:50.000000000 +0100
@@ -263,7 +263,7 @@ static void draw_bar_scale(int* y) {
             char s[40], *p;
             int x;
             /* This 1024 vs 1000 stuff is just plain evil */
-            readable_size(i, s, sizeof s, options.log_scale ? 1000 : 1024, 0);
+            readable_size(i, s, sizeof s, options.log_scale ? 1000 : 1024, options.bandwidth_in_bytes);
             p = s + strspn(s, " ");
             x = get_bar_length(i * 8);
             mvaddch(*y + 1, x, ACS_BTEE);
